src/data/WorkshopSchedule.js

```diff
@@ -1,75 +1,186 @@
-import React from 'react';
-
-const hothDate = '2021-02-23'; // replace with real date later
+const hothDate = '2021-01-'; // replace with real date later
 const utcOffset = '-08:00'; // PST
 // time is a string of the form "HH:mm:ss"
 function getDateObj(time) {
-	return new Date(`${hothDate}T${time}${utcOffset}`);
+	return new Date(`${hothDate}${time}${utcOffset}`);
```

Comment:
If you can, just modify the function to also take in the day (i.e. `05`, `06`, etc.), and rewrite the function body to mimic the original. The new function should take a `day` and a `time` parameter. (Likewise, modify the places where you called the function to pass in two arguments).

While the solution you came up with works and is very clever, it creates a leak in the abstraction, as the user has to know how this function is implemented to use it.